ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -2648,7 +2648,7 @@ public sealed class ArchiWebHandler : IDisposable {
 			Steam.UserPrivacy userPrivacy;
 
 			try {
-				userPrivacy = JsonConvert.DeserializeObject<Steam.UserPrivacy>(json);
+				userPrivacy = JsonConvert.DeserializeObject<Steam.ProfileEditData>(json).Privacy;
```

Comment:
This will fail with NRE if the object is null. You must check userPrivacy against null below, and if it isn't null, take `Privacy` from it.

Reply:
And you should also check if privacy isn't null, probably with something like `userPrivacy?.Private` to ease you on one more null check.